src/System.Collections/src/System/Collections/Generic/SortedSet.TreeSubSet.cs

```diff
@@ -124,6 +124,54 @@ internal override bool IsWithinRange(T item)
                 return comp >= 0;
             }
 
+            internal override T MinInternal
+            {
+                get
+                {
+                    Node current = _root;
+                    T result = default(T);
+
+                    while (current != null) {
+
+                        int comp = _lBoundActive ? Comparer.Compare(_min, current.Item) : -1;
+                        if (comp == 1) {
+                            current = current.Right;
+                        } else {
+                            result = current.Item;
+                            if (comp == 0)
+                                break;
+                            current = current.Left;
+                        }
+                    }
+
+                    return result;
+                }
+            }
```

Comment:
Nits:
- Could you move the opening braces from the end of the lines to the beginning of the next?
- Could you move the else to its own line?
- Could you surround the break with braces (on their own lines)?

Same in MaxInternal below.